device/thunder_ripple_sdk/src/client/device_operator.rs

```diff
@@ -91,6 +91,30 @@ impl DeviceChannelRequest {
             }
         }
     }
+
+    pub fn get_dev_call_request(&self) -> Option<DeviceCallRequest> {
```

Comment:
If it is used for mock, prepend `mock_xxx` in the function name and include this under test attribute.